wandb/sdk/internal/internal_api.py

```diff
@@ -1706,7 +1707,14 @@ def create_launch_agent(
                 $agentConfig: JSONString"""
             mutation_input += """,
                         agentConfig: $agentConfig"""
+        if "version" in self.create_launch_agent_fields_introspection():
```

Comment:
is this established style in this repo? whitespace/format is totally optional in gql and imo this would be much easier to read w/o the extra commas and newlines.

this is totally valid: `mutation foo($x: String) { bar { baz } qux }`

if we still want a pretty formatted payload, we can use `print_ast`:

```
>>> import gql
>>> from graphql.language import print_ast
>>> print_ast(gql.gql('mutation foo($x: String) { bar { baz } qux }'))
'mutation foo($x: String) {\n  bar {\n    baz\n  }\n  qux\n}'
```

Reply:
> whitespace/format is totally optional in gql

TIL, thanks. This simplifies a lot.